vehicle_interface_node:
  ros__parameters:
    NX: 5                 # number of states currently : x,y,yaw,vx,steering
    NU: 2                 # control acc,steering_dot
    integration_deltaT: 0.001     #  rk4/euler integration time-step
    sim_deltaT: 0.01             # state update time
    state_publish_deltaT: 0.05  # state publisher delta-time
    veh_width: 0.31           # in meters
    veh_length: 0.58   
    veh_wheel_radius: 0.35
    veh_wheelbase: 0.34
    mass: 1200.0         # in kg
    default_x_pos: 0.0
    default_y_pos: 0.0
    default_yaw: 0.0
    default_vx: 0.0
    default_sf: 0.0
    default_acc: 0.0
    default_sv:  0.0
    map: "e7_floor5_square.csv"

visualizer_node:
  ros__parameters:
    NX: 5                 # number of states currently : x,y,yaw,vx,steering
    NU: 2                 # control acc,steering_dot
    integration_deltaT: 0.001     #  rk4/euler integration time-step
    sim_deltaT: 0.01             # state update time
    state_publish_deltaT: 0.05  # state publisher delta-time
    veh_width: 0.31           # in meters
    veh_length: 0.58         # in meters
    veh_wheel_radius: 0.35
    veh_wheelbase: 0.34
    mass: 1200.0         # in kg
    default_x_pos: 0.0
    default_y_pos: 0.0
    default_yaw: 0.0
    default_vx: 0.0
    default_sf: 0.0
    default_acc: 0.0
    default_sv:  0.0
    map: "e7_floor5_square.csv"
    plot_update_dt: 0.05 #plotting time
    x_lim: 20.0
    y_lim: 20.0
